README.md

```diff
@@ -25,10 +25,12 @@ A Flask application run in AWS Lambda used to process invoices from Alma. Invoic
 
 ```shell
 ALMA_SAP_INVOICES_ECR_IMAGE_NAME=### The name of the ECR image for the Alma SAP Invoices CLI.
-ALMA_SAP_INVOICES_ECS_CLUSTER=### The short name or full ARN of the ECS cluster to run the Alma SAP Invoices CLI ECS task.
+ALMA_SAP_INVOICES_ECS_CLUSTER=### The full ARN of the ECS cluster to run the Alma SAP Invoices CLI ECS task.
 ALMA_SAP_INVOICES_ECS_TASK_DEFINITION=### The family and revision (formatted as <family>:<revision>) or full ARN of Alma SAP Invoices CLI ECS task. 
 ALMA_SAP_INVOICES_ECS_NETWORK_CONFIG=### The network configuration for the Alma SAP Invoices ECS task.
 ALMA_SAP_INVOICES_CLOUDWATCH_LOG_GROUP=### The name of the log group containing logs from Alma SAP Invoices ECS task runs.
+LOGIN_DISABLED=### String variable representing a boolean to turn off authentication (i.e., ignore 'login_required' decorator). Set to 'true' to disable logging (recommended for unit testing and Dev); must be set to 'false' for Stage and Prod.
```

Comment:
Just clarifying, should this be `Set to 'true' to disable authentication (recommended...`?

Reply:
Oof, good eye! Let me fix this. See [updated description](https://github.com/MITLibraries/alma-sapinvoices-ui/pull/60/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R32). I wanted to differentiate "authentication" from "login". For this app, login = identifying the user. 🤔